src/components/Buttons/Google.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Button } from "@mui/material";
+import styles from '../Buttons/buttons.module.scss'
+
+const Google = () => {
```

Comment:
Lo ideal de React es hacer un componente que se adapte a todo tipo de situaciones. Este boton se podria combinar con todos los de la carpeta de Buttons y te ahorras Github.js, Google.js, IniciarSesion.js .
```suggestion
const Google = () => {
return (
    <div className={`${styles.container}`}>
        <Button variant="outlined" className={`${styles[className]}`}>{text}</Button>
    </div>
  )
}
export default Google
```